application/main.go

```diff
@@ -237,7 +238,8 @@ func get_zmq_updates(connectAddr string, regManager *cj.RegistrationManager, con
 
 				if !reg.PreScanned() {
 					// New registration received over channel that requires liveness scan for the phantom
-					liveness, response := reg.PhantomIsLive()
+					liveness, response := regManager.PhantomIsLive(reg.DarkDecoy, 443)
```

Comment:
`reg.DarkDecoy` is type `net.IP` so this either needs a to be `reg.DarkDecoy.String()` or we could update the `PhantomIsLive` function to take net.IP as an argument so that we aren't stringifying an IP just to re-parse it. Up to you.